src/Search.js

```diff
@@ -41,27 +51,169 @@ export default function Search() {
     <Container>
       <Row>
         <Col>
+          <Container>
+            {showAlert && (
+              <Alert variant='info' onClose={() => setShowAlert(false)} dismissible>
+                <Alert.Heading>Usage</Alert.Heading>
+                <p>
+                  Click on the list in the right to add a card, use the selection above to filter
+                  the cards in the list
+                </p>
+              </Alert>
+            )}
+          </Container>
+
           <Form>
             <InputGroup className='my-3 mb-3 text-light'>
-              <Form.Control
-                style={{ borderColor: '#444' }}
-                className='bg-dark text-light'
-                onChange={(e) => setSearch(e.target.value)}
-                placeholder='Search'
-                value={search}
-              ></Form.Control>
               <Form.Select
                 style={{ borderColor: '#444' }}
                 className='bg-dark text-light'
                 value={quality}
                 onChange={(e) => setQuality(parseInt(e.target.value))}
+                isInvalid={quality == 0}
```

Comment:
`===` is better than `==`

With 3 = symbols, you are checking also the type of the variable, because

![image](https://user-images.githubusercontent.com/2400723/218281824-10e02b3f-b878-4ff2-af2e-d9bd14d41386.png)

but

![image](https://user-images.githubusercontent.com/2400723/218281835-800defe2-30bc-42b5-adc1-40a0faabf6c8.png)